client/post-editor/media-modal/index.jsx

```diff
@@ -15,7 +15,7 @@ var MediaLibrary = require( 'my-sites/media-library' ),
 	PostActions = require( 'lib/posts/actions' ),
 	PostStats = require( 'lib/posts/stats' ),
 	MediaModalSecondaryActions = require( './secondary-actions' ),
-	MediaModalDetail = require( './detail' ),
+	MediaModalDetail = require( './detail' ).default,
```

Comment:
Could we change this to an import below. This will also break if we only export one thing from MediaModalDetail